src/bcrhp/bcrhp/migrations/909_update_url_prefix.py

```diff
@@ -0,0 +1,542 @@
+# Generated by Django 4.2.13 on 2024-07-05 09:19
+
+from django.db import migrations
+from bcrhp.migrations.operations.privileged_sql import RunPrivilegedSQL
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('bcrhp', '947_historic_site_one_line'),
+    ]
+
+    operations = [
+        RunPrivilegedSQL(
+            """
+            begin;
+            update map_layers
+            set legend = replace(legend, '/int/bcrhp/', '/bcrhp/');
+            commit;
+            """,
+            """
+            begin;
+            update map_layers
+            set legend = replace(legend, '/bcrhp/', '/int/bcrhp/');
+            commit;
+            """
+        ),
+        RunPrivilegedSQL(
```

Comment:
The RunPrivilegedSQL is only required when making DDL changes. Data updates should be done with RunSQL as the default user has access to modify data.

Reply:
Updated all the data updates and just left the v_historic_site using the priv ops